src/utils/constants.ts

```diff
@@ -7,3 +7,9 @@ export const PLUGIN_PERMISSION_DELEGATE_CALL = 4n;
 export const MODULE_TYPE_PLUGIN = 1n;
 export const MODULE_TYPE_FUNCTION_HANDLER = 2n;
 export const MODULE_TYPE_HOOKS = 4n;
+export const MODULE_TYPE_SIGNATURE_VALIDATOR_HOOKS = 8n;
+export const MODULE_TYPE_SIGNATURE_VALIDATOR = 16n;
```

Comment:
food-for-thought: Not related to this PR, but since the module type is a `uint8`, might make sense to represent them as `number` on the TypeScript side (AFAIK, `ethers` will happily convert `number` to `bigint` when needed).

Reply:
https://github.com/safe-global/safe-core-protocol/pull/109/commits/b011afca62c4e56e28cb16d29d8e8fefeec98534

Reply:
Oh nice, fixed it here and now 💪 - I really meant it as "here is something I noticed when reviewing that can be addressed in a future PR", but also nice that its in already :)